requirements.txt

```diff
@@ -1,4 +1,4 @@
 twilio==5.4.0
 mock>=1.0
+djangorestframework<=4.0
```

Comment:
Shouldn't this be the same as in setup.py?

Reply:
Yes, that's right